keyboards/ergodash/rev1/keymaps/333fred/config.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include QMK_KEYBOARD_CONFIG_H
+#include "333fred_config.h"
```

Comment:
If you named that file just `config.h`, it should get picked up automatically.

Reply:
Yeah, back when that was created it was just you, Jack, and me with userspaces and I don't think we had that autoinclude. I'll refractor it eventually.

Reply:
Yeah, we didn't.  It was added later, because, well.... it should have been there, and it was a pain that it wasn't.